app/views/unattended/provisioning_templates/registration/global_registration.erb

```diff
@@ -96,22 +106,36 @@ else
 fi
 <% end -%>
 
+<%
+if @download_utility == 'curl'
+  data_keyword = "--data"
+else
+  data_keyword = "--post-data"
+end
+
+if @download_utility == 'curl'
+  registration_command="curl --silent --show-error --request POST --cacert"
+else
+  registration_command="wget --no-verbose -O- --ca-certificate"
+end
```

Comment:
Perhaps group the together?
```suggestion
if @download_utility == 'curl'
  data_keyword = "--data"
  registration_command="curl --silent --show-error --request POST --cacert"
else
  data_keyword = "--post-data"
  registration_command="wget --no-verbose -O- --ca-certificate"
end
```

Or another way:
```ruby
def build_command(utility, url, headers=[], data={})
  # ...
end
```

Then you can use it in both places.